ql/src/java/org/apache/hadoop/hive/ql/exec/repl/incremental/IncrementalLoadTasksBuilder.java

```diff
@@ -266,16 +271,15 @@ private boolean shouldReplayEvent(FileStatus dir, DumpType dumpType, String dbNa
     return updateReplIdTxnTask;
   }
 
-  private Task<? extends Serializable> tableUpdateReplStateTask(String dbName, String tableName,
-                                                    Map<String, String> partSpec, String replState,
-                                                    Task<? extends Serializable> preCursor) throws SemanticException {
+  private Task<? extends Serializable> tableUpdateReplStateTask(TableName tableName, Map<String, String> partSpec,
```

Comment:
now that I see this...I think that it would probably make sense to also include `partSpec` into this object
(`PartishRef`: `catalog,database,tablename,partitonspec` ) ?

Reply:
Besides a greater level of enforcement, is there any other benefit?
Inside `org.apache.hadoop.hive.ql.plan` only `ShowLocksDesc` is using both tableName, dbName and partitionspec, the other Desc classes that have partitionspec only use tableName.